cmd/tailscale/cli/exitnode.go

```diff
@@ -202,23 +202,20 @@ type filteredCity struct {
 	Peers []*ipnstate.PeerStatus
 }
 
-const noLocationData = "-"
+const noLocationData = ""
```

Comment:
maybe just nuke this constant now and change callers to say "" ?

Reply:
Agreed. I also removed this global that nobody was directly comparing against:
``` go
var noLocation = &tailcfg.Location{
	Country:     noLocationData,
	CountryCode: noLocationData,
	City:        noLocationData,
	CityCode:    noLocationData,
}
```